### ISSUE TITLE
Proben für einzelne Instrumente

### ISSUE THREAD

**Issue by ULinn** [mattimaier/bnote#383]
Statt bei einer neuen Probe nur eine Gruppe zuordnen zu können, wäre es in manchen Fällen (Satzprobe, Einzelprobe etc.) sinnvoller, eine Zuordnung nach Instrument vornehmen zu können. Hier sollte eine Mehrfach-Auswahl möglich sein - z.B. Tenor 1, Tenor 2, Bass 1, Bass 2 für eine "Männer-Probe".

Noch besser wäre es, wenn zusätzlich noch einzelne Personen hinzugefügt werden könnten (Solist, Techniker o.ä.).

**Comment by mattimaier** [comment]
Letzteres ist kein Problem:
Probe > auswählen > Teilnehmer > Einladung hinzufügen

**Comment by ULinn** [comment]
Stimmt, danke für den Hinweis, bleibt nur noch die Einladung nach Instrument.
